src/guides/v2.4/graphql/mutations/add-products-to-cart.md

```diff
@@ -0,0 +1,324 @@
+---
+group: graphql
+title: addProductsToCart mutation
+redirect from:
+  - /guides/v2.4/graphql/mutations/add-bundle-products.html
+  - /guides/v2.4/graphql/mutations/add-configurable-products.html
+  - /guides/v2.4/graphql/mutations/add-downloadable-products.html
+  - /guides/v2.4/graphql/mutations/add-simple-products.html
+  - /guides/v2.4/graphql/mutations/add-virtual-products.html
+---
+
+In order to simplify the flow of adding different types of products to a cart, it is proposed to use a single mutation for all product types.
+
+We implemented the single mutation "AddProductsToCart" from Magento 2.4.1.
+
+The `addProductsToCart` mutation allows you to add any number of products to the cart at the same time.
+
+To add products to a cart, you must provide the cart ID, the SKUs, and the quantities. You can also optionally provide customizable options.
```

Comment:
You must specify the Cart ID along with the list of SKU and Quantity pairs as parameters to add the products to the shopping cart.